sdk/modelsrepository/azure-iot-modelsrepository/src/main/resources/azure-digital-twins.properties

```diff
@@ -0,0 +1,2 @@
+name=${project.artifactId}
```

Comment:
File should be renamed

Reply:
From the client builder class, it should be named "azure-iot-modelsrepository.properties"

```java
private static final String MODELS_REPOSITORY_PROPERTIES = "azure-iot-modelsrepository.properties";
```

Reply:
Obviously :D